nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -70,12 +70,12 @@ NavfnPlanner::configure(
 
   // Initialize parameters
   // Declare this plugin's parameters
-  declare_parameter_if_not_declared(node_, "tolerance", rclcpp::ParameterValue(0.0));
-  node_->get_parameter("tolerance", tolerance_);
-  declare_parameter_if_not_declared(node_, "use_astar", rclcpp::ParameterValue(false));
-  node_->get_parameter("use_astar", use_astar_);
-  declare_parameter_if_not_declared(node_, "allow_unknown", rclcpp::ParameterValue(true));
-  node_->get_parameter("allow_unknown", allow_unknown_);
+  declare_parameter_if_not_declared(node_, name + ".tolerance", rclcpp::ParameterValue(2.0));
```

Comment:
Is this 2.0 meters to goal? Why?

Reply:
It was `0.0` as default before, which is not necessarily possible. Having a lower tolerance lets a robot navigate to a close-by pose if the pose requested is actually occupied.

This tolerance isn't used for planning on the first run, it will try to get to an exact point. What this is useful for is the ability to get close to the goal when the goal requested is actually invalid. 2.0m is pretty reasonable I think.

Also, there's some updates here, so please don't merge this, I found in writing and making experiments for the RSS paper. Nothing fundamentally changes but there were some performance issues with some of the parameters set by default in this change with NavFn & DWB

Reply:
So if it can navigate to the goal, it will, but if it can't it will keep trying at incremental distances away from the goal up to 2.0 meters?

Reply:
Yup!